swift/extractor/main.cpp

```diff
@@ -3,15 +3,58 @@
 #include <stdlib.h>
 
 #include <swift/Basic/LLVMInitialize.h>
+#include <swift/FrontendTool/FrontendTool.h>
+
+#include "Extractor.h"
+
+using namespace std::string_literals;
+
+class Observer : public swift::FrontendObserver {
```

Comment:
Any suggestions what should be included in the doc?

Reply:
1. What it does (triggers extraction) and
2. Why we need it (because that's the API the Swift frontend provides).

Something along the lines of "Wraps the Swift extractor in a visitor for use with Swift's frontend API." - if that's an accurate understanding.

Reply:
@sashabu added a comment.